### ISSUE TITLE
Engintron Dedicated IP issue

### ISSUE THREAD

**Issue by fastwebhost** [engintron/engintron#274]
Hi,

Sites with dedicated IP's are not working when we enable engintron in server. We have tried below rules in default.conf file.
===
if ($host ~ "example.com") {
    set $PROXY_DOMAIN_OR_IP "XXX.XXX.XXX.XXX";
}
===
If there are any alternative options to exclude domain from engintron please let us know.

**Comment by fevangelou** [comment]
You should not add this in default.conf but in custom_rules. Try again, restart Apache/Nginx and let me know.

**Comment by fastwebhost** [comment]
Hi,

We have added in the custom rules and the issue fixed. But the thing we need to add each and every domain in custom rules who are using the dedicated IP.

**Comment by katmai** [comment]
yeah i sadly had to remove engintron from a bunch of servers. i believe the majority of serious providers have a server with many clients and quite a few dedicated ips. it's insanely painful to add accounts with like 20 30 addon domains and dedicated ip in custom rules, especially if you have like 100 200 300 domains per server.

**Comment by JorgeMarcelino-HS** [comment]
Hello, any news about it?

**Comment by Leykis** [comment]
@katmai Checkout this script someone has written to help with this process : https://github.com/engintron/engintron/wiki/3rd-party-scripts-for-Engintron

**Comment by katmai** [comment]
nice. i will check it out

**Comment by fevangelou** [comment]
You don't need to specify the IP for such domains inside "custom rules". This is just a way to enforce Nginx's resolving to a given IP.

Of course if you can troubleshoot further and provide snapshots of the access and error logs of your server, it will be easier for me to pinpoint why some servers require the use of if blocks for domains with dedicated IPs.
